File: proyecto_java/ProyectoFinalJava/src/proyectofinaljava/Reporte.java
package proyectofinaljava;

public class Reporte {

    public static String obtenerInformacion(int actividadesRegistradas) {
        if (actividadesRegistradas == 0) {
            return "Mala campaña, debemos mejorar";
        } else {
            if (actividadesRegistradas >= 1 && actividadesRegistradas <= 5) {
                return "Poca participación en el club, hay que mejorar";
            } else {
                if (actividadesRegistradas >= 6 && actividadesRegistradas <= 15) {
                    return "Buena participación, sigan así";
                } else {
                    return "Excelente campaña del club";
                }
            }
        }
    }

    public static String obtenerReporte(String[] actividades, int[] participantes) {
        String reporte = "Las actividades ingresadas son:\n";
        for (int i = 0; i < actividades.length; i++) {
            reporte = String.format("%s%d. Número de Participantes de %s: %d\n",
                    reporte, (i + 1), actividades[i], participantes[i]);
        }
        return reporte;
    }
}

File: proyecto_python/reporte.py
def obtener_informacion(actividades_registradas):
    if actividades_registradas == 0:
        return "Mala campaña, debemos mejorar"
    elif 1 <= actividades_registradas <= 5:
        return "Poca participación en el club, hay que mejorar"
    elif 6 <= actividades_registradas <= 15:
        return "Buena participación, sigan así"
    else:
        return "Excelente campaña del club"

def obtener_reporte(actividades, participantes):
    reporte = "Las actividades ingresadas son:\n"
    for i in range(len(actividades)):
        reporte += f"{i+1}. Número de Participantes de {actividades[i]}: {participantes[i]}\n"
    return reporte